src/frontend/web_application/src/components/ContactAvatarLetter/index.jsx

```diff
@@ -29,10 +29,12 @@ const ContactAvatarLetter = ({ contact, size, isRound }) => {
 
   const letterClassNameSize = classNameSize ? `${classNameSize}__letter` : null;
 
+  const letterClassName = className ? `${className}__letter` : null;
```

Comment:
ok! finally find a very simple way to solve issue (font-size and line-height on m-avatar instead of m-avatar__letter)